packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -12,9 +12,12 @@ Generated by [AVA](https://avajs.dev).
     ␊
     Run Netlify Build system locally.␊
     ␊
-    Options can also be specified as environment variables prefixed with␊
-    NETLIFY_BUILD_. For example the environment variable NETLIFY_BUILD_DRY=true can␊
-    be used instead of the CLI flag --dry.␊
+    Options can also␊
+    be specified as environment variables prefixed with␊
+    NETLIFY_BUILD_. For example␊
+    the environment variable NETLIFY_BUILD_DRY=true can␊
+    be used instead of the CLI f␊
+    lag --dry.␊
```

Comment:
This seems to be a bug with the new version of Yargs. I have noticed in some other projects too.

Reply:
Adding random line breaks? Should we update it then? as it will break the documentation?

Reply:
Hum, good question.
I don't think this is too much of a problem because our users are not directly using `@netlify/build` and `@netlify/config`'s binaries. Those are used inside the buildbot, but not the `--help` CLI flag. Netlify CLI and our users are using the programmatic interface otherwise.